src/test/java/harness/baseclasses/SingleWorkspaceUnitGcp.java

```diff
@@ -0,0 +1,12 @@
+package harness.baseclasses;
+
+import bio.terra.workspace.model.CloudPlatform;
+import java.util.Optional;
+
+/** Base class for GCP unit tests that only need a single workspace for all test methods. */
+public class SingleWorkspaceUnitGcp extends SingleWorkspaceUnit {
```

Comment:
nit: singleGcpWorkspaceUnit

Reply:
Kept it as SingleWorkspaceUnit<Platform> so that it is easier to find and maintain